src/main/scala/cromwell/engine/package.scala

```diff
@@ -52,6 +52,7 @@ package object engine {
     }
 
     val backend = Backend.from(workflowOptions.getOrElse("default_backend", ConfigFactory.load.getConfig("backend").getString("backend")))
+    val avoidJobs = ConfigFactory.load.getConfig("jobavoidance").getBoolean("enabled")
```

Comment:
Also, instead of "jobavoidance", perhaps camel case or hyphenation? Not sure what's standard for cromwell. For use with various other APIs there seem to be both in application.conf right now.